eqllib/analytics/discovery/T1087-discovery-commands-koadic.toml

```diff
@@ -0,0 +1,25 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies initial system enumeration and discovery commands tied to Koadic's process ancestry."
```

Comment:
This is still fairly tool specific. More by description than by design. I wonder if we can make a generic "RunDLL32 Remote Access Tool" detection out of this. Network + Enumeration command. What do you think of this?

```sql
sequence with maxspan=1h
  [process where subtype.create and process_name == "rundll32.exe"] by unique_pid
  [network where subtype.outgoing and process_name == "rundll32.exe"] by unique_pid
  [process where subtype.create and parent_process_name == "rundll32.exe"] by unique_ppid
```

Reply:
I dig this, it's a lot more simpler.  It won't match the blog post, but I am fine with that as it will make it more broad. I will adjust the description.